tests/integration/single_table/test_copulas.py

```diff
@@ -365,14 +365,11 @@ def test_numerical_columns_gets_pii():
     """Test that the synthesizer works when a ``numerical`` column gets converted to ``PII``."""
     # Setup
     data = pd.DataFrame(data={
-        'id': [0, 1, 2, 3, 4],
         'city': [0, 0, 0, 0, 0],
         'numerical': [21, 22, 23, 24, 25]
     })
     metadata = SingleTableMetadata.load_from_dict({
-        'primary_key': 'id',
         'columns': {
-            'id': {'sdtype': 'id'},
```

Comment:
why are we removing these?

Reply:
I removed it since it'll just be randomly generated numbers and I couldn't think of a good way to add an `assert` around them. I could change them to non-numerical and check the `sdv-id-` format instead?

Reply:
Ah I see. Yeah that's fine. I thought we controlled faker's randomness with a seed though

Reply:
Ah yeah, I can add it back then if we want to keep the ids

Reply:
Maybe we should so that we know if the randomness isn't being controlled